src/main/java/interface_adapters/RescheduleAutoController.java

```diff
@@ -0,0 +1,7 @@
+package main.java.interface_adapters;
+
+import main.java.use_case.TaskInfo;
+
+public interface RescheduleAutoController {
+    boolean rescheduleTime(TaskInfo task);
```

Comment:
This method should take in `EventInfo` instead of `TaskInfo` since we are rescheduling an existing event